test/send-to-fluentd/20001-fake-1-logging-udp.py

```diff
@@ -0,0 +1,21 @@
+import testlib
+import time
+
+testlib.send_logs.udp_connect('localhost', 20001)
```

Comment:
change to 20002 for udp?

Reply:
Ok, and I plan to do another http test script for port 20003 later.